fe/fe-core/src/test/java/com/starrocks/analysis/ShowRoutineLoadStmtTest.java

```diff
@@ -0,0 +1,55 @@
+// This file is made available under Elastic License 2.0.
+// This file is based on code available under the Apache license here:
```

Comment:
The Apache license is not necessary, since it's the newly created file

Reply:
[use the es license](https://github.com/StarRocks/starrocks/pull/8909/commits/b0bde31eb14f8054d69f85fcc05f1a75103a7f90)